src/dlls/engine/7_newday/newday.c

```diff
@@ -194,32 +208,36 @@ void dll_7_func_DAC(s32 *param1) {
 }
 
 /** 
-  * Checks if nighttime
-  * Also sets the argument float to the seconds until nighttime/daytime 
-  */
-s32 dll_7_func_DF4(f32* timeSeconds) {
+ * offset: 0xDF4 | func: 8 | export: 8
+ * Checks if nighttime
+ * Also sets the argument float to the seconds until nighttime/daytime
+ */
+s32 dll_7_set_seconds_till_nighttime(f32* outTime) {
```

Comment:
this func name is a little confusing, it's not setting anything (aside from the out param). i don't really know what to call it tho